content/md/products-super-power/enrich-products-variants.md

```diff
@@ -38,25 +38,59 @@ If the background of the `Variant products` is:
 
 ## Filter on your attributes
 
-To easily manage the products with variants and their until 3 levels of enrichment, we have added a **smart** search in the grid for products with variants to automatically display what you are looking for.
+To easily manage the products with variants up to 3 levels of enrichment, we added a **smart** search in the grid for products with variants to automatically display what you are looking for. Either a product model, or a product.
```

Comment:
isn't it the otther way round: "to easily manage the up to 3 levels of enrichment of procdcts with variant"? Caill, we miss you!!!

Reply:
Updated with the 2.0 release note sentence, validated by our teacher Camille ;)